README.md

```diff
@@ -62,18 +62,9 @@ The storage account sample is broken down into 8 different methods called in seq
     // Demonstrate regeneration of a storage account key
     await regenerateStorageAccountKey(storageAccount, vault);
     
-    // Demonstrate listing off the storage accounts in the vault
-    await getStorageAccounts(vault);
-    
     // Demonstrate the creation of an account-level SAS definition 
     await createAccountSASDefinition(storageAccount, vault);
     
-    // Demonstrate the creation of a container-level SAS definition
-    await createBlobSASDefinition(storageAccount, vault);
-    
-    // List all SAS definitions in the account
-    await getSASDefinitions(storageAccount, vault);
-    
```

Comment:
why do we remove these functions ?

Reply:
The latest SDK has no such implementation. The change to the sample is a little big, because  the storage account in `Track1` is managed through the keyvault client .

Reply:
I see, let me check